docs/dapp/tools/band.md

```diff
@@ -17,7 +17,7 @@ for full list of deployed contract addresses.
 :::
 
 [paratime]: /general/oasis-network/faq#how-is-a-paratime-different-from-a-parachain
-[emerald]: /dapp/emerald
+[emerald]: /dapp/tools/other-paratimes/emerald/README.mdx
```

Comment:
```suggestion
[emerald]: ../other-paratimes/emerald/README.mdx
```

Please also fix the paratime URL above to
```
../../general/oasis-network/faq.md#how-is-a-paratime-different-from-a-parachain
```
so that it works when viewing it outside of our docs.